### ISSUE TITLE
Running plantuml-server with gollum

### ISSUE THREAD

**Issue by ace03uec** [gollum/gollum#1017]
I have a gollum instance, running with a plantuml server running on the same instance. I have added the config given on the Plantuml wiki page to config.rb

```ruby 
 Gollum::Filter::PlantUML.configure do |config|
      config.url = "http://localhost:8080/plantuml/png"
 end 
```

 and when i try to run the gollum instance, I get the following error

```shell
/home/user/config.rb:25:in `<top (required)>': uninitialized constant Gollum::Filter::PlantUML (NameError)
	from /usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/share/ruby/gems/2.0/gems/gollum-4.0.0/bin/gollum:201:in `<top (required)>'
	from /usr/local/bin/gollum:23:in `load'
	from /usr/local/bin/gollum:23:in `<main>'
```

The Plantuml server seems to be working fine. Can access it and it generates the UML's correctly. Am I missing some configuration ?

**Comment by dometto** [comment]
On 27-05-15 13:45, Giridaran Manivannan wrote:
> and when i try to run the gollum instance, I get the following error
> /home/user/config.rb:25:in  `<top (required)>': uninitialized constant Gollum::Filter::PlantUML (NameError)
>      from /usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
>      from /usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in  `require'
>      from /usr/local/share/ruby/gems/2.0/gems/gollum-4.0.0/bin/gollum:201:in `<top (required)>'
>      from /usr/local/bin/gollum:23:in  `load'
>      from /usr/local/bin/gollum:23:in `<main>'
>
> The Plantuml server seems to be working fine. Can access it and it 
> generates the UML's correctly. Am I missing some configuration ?
>
I think PlantUML is only available in the `rc` branch of gollum-lib. I 
was waiting for some other milestones before merging that into `master`, 
that's why it's been delayed (and why it's already in the wiki). Sorry 
about that. As a workaround, I think you can probably get away with 
installing gollum-lib from git, using the `rc` branch.

We'll have to decide whether it's a good idea to merge the `rc` branch 
as-is. @bartkamphorst, @sunny?

**Comment by sunny** [comment]
I am sorry, I don't know what other milestones need to get merged in yet as I haven't been following developments. I'd suggest to ship whatever is ready as soon as you can.

I am closing this

**Comment by dometto** [comment]
@ace03uec, I just released gollum-lib v4.1.0, which contains PlantUML support. Updating should fix the problem for you!
